doc/releases/changes.rst

```diff
@@ -15,6 +15,16 @@ Bug Fixes
 Backward-Incompatible Changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Version 5.1.2 (released Oct 4, 2023)
+----------------------------------------
+
+- Update Vega-Lite from version 5.14.1 to version 5.15.1;
+  see `Vega-Lite Release Notes <https://github.com/vega/vega-lite/releases>`_.
```

Comment:
```suggestion
  see `Vega-Lite Release Notes <https://github.com/vega/vega-lite/releases>`_.
- Updated vl-convert from version 0.13.0 to version 0.14.0;
  see `vl-convert Release Notes <https://github.com/vega/vl-convert/releases>`_.
  - Notably, this release makes it possible to save Altair charts as PDFs via `chart.save('filename.pdf')`
```

@jonmmease I realize we're highlighting the bumps in Vega-Lite version and was thinking maybe we should do the same for VegaFusion and vl-convert? At least when they add new features, such as 0.14 adding support for saving charts as PDFs, which is something I definitely think we should highlight in the release notes for Altair somehow.

Reply:
Ok that sounds great. Having a 5.2 release for this improvement feels appropriate!